metering/metering-uninstall.adoc

```diff
@@ -1,13 +1,27 @@
 :context: metering-uninstall
 [id="metering-uninstall"]
-= Uninstalling Metering
+= Uninstalling metering
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 You can remove metering from your {product-title} cluster.
 
-include::modules/metering-uninstall.adoc[leveloffset=+1]
+[NOTE]
+====
+Metering does not manage or delete Amazon S3 bucket data. You must manually clean up S3 buckets used to store metering data.
+====
+
+[id="metering-remove"]
+== Removing the Metering Operator from your cluster
 
+Remove the Metering Operator from your cluster by following the documentation on xref:../operators/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
 
+[NOTE]
+====
+Removing the Metering Operator from your cluster does not remove its custom resource definitions or managed resources. See the following sections on "Uninstalling metering from your namespace" and "Uninstalling metering CustomResourceDefinitions" for steps to remove any remaining metering components.
```

Comment:
Since you added this to the assembly, you can use xrefs to link to the below sections.